x-pack/test_serverless/functional/page_objects/svl_search_connectors_page.ts

```diff
@@ -86,7 +86,7 @@ export function SvlSearchConnectorsPageProvider({ getService }: FtrProviderConte
             .catch(() => false)
         );
         browser.refresh();
```

Comment:
Maybe even this should be awaited? It's an async call - by not awaiting this one, we're putting the time it should take to reload the page on the timeout headroom of the expectation, risking further timeouts.
```suggestion
        await browser.refresh();
```

Reply:
Yes. it makes sense. It's my mistake, I missed to add await here.